### PR TITLE
Feature/delete todos

### PR DESCRIPTION
**Pull request by yannhn**

### CODE REVIEW

File: src/components/ToDos/ToDo/index.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import {ToDoItemContainer} from './styled';
 
-export default function ToDo({title}) {
+export default function ToDo({title, deleteToDo}) {
 	return (
 		<ToDoItemContainer>
 			<input type="checkbox"></input>
```
Comment (ahohnsen) [inline src/components/ToDos/ToDo/index.js:6]: I suggest to add a label here as each input element should have one :)

File: src/components/ToDos/ToDo/index.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 			<p>{title}</p>
```
Comment (ahohnsen) [inline src/components/ToDos/ToDo/index.js:7]: This could be the label instead of a `p`-tag.

File: src/components/ToDos/ToDoList/index.js [hunk 3]
```diff
@@ -16,12 +16,17 @@ export default function ToDoList() {
 		});
 	}
 
+	function deleteToDo(id) {
+		const filteredItems = items.filter(item => id !== item.id);
```
Comment (ahohnsen) [inline src/components/ToDos/ToDoList/index.js:20]: I suggest renaming your `state` and this `const` here (e.g. ToDos).

**Review by ahohnsen — CHANGES_REQUESTED** [review]
Great job! I have a few suggestions regarding accessibility and naming :)

**Comment by yannhn** [comment]
Thanks for the feedback @ahohnsen ! I already implemented it.

**Pull request merged**